ddl/sequence_test.go

```diff
@@ -153,3 +153,30 @@ func (s *testSequenceSuite) TestDropSequence(c *C) {
 	c.Assert(err, NotNil)
 	c.Assert(err.Error(), Equals, "[planner:1142]DROP command denied to user 'localhost'@'myuser' for table 'my_seq'")
 }
+
+func (s *testSequenceSuite) TestSequenceAsDefaultValue(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use test")
+	s.tk.MustExec("create sequence seq")
+
+	// test the use sequence's nextval as default
+	_, err := s.tk.Exec("create table t (a int default next value for seq)")
+	c.Assert(err, IsNil)
```

Comment:
Please check the error code or error msg.

Reply:
Here the error is nil, before the PR this will crash

Reply:
Here the error is nil, before the PR this will crash.

Reply:
Sorry. Ignore it.